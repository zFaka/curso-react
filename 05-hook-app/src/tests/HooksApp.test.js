import {shallow} from 'enzyme';
import React from 'react';
import HookApp from '../HooksApp';

describe('Test HookApp', () => {

  test('Testing component', () => {
    const wrapper = shallow(<HookApp/>);

    expect(wrapper).toMatchSnapshot();
  })

})
