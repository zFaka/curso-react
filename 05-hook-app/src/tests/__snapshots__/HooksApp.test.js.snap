// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HookApp Testing component 1`] = `
<Fragment>
  <h1>
    Hola mundo
  </h1>
</Fragment>
`;
